doc/getting_started/setup_fpga.md

```diff
@@ -257,16 +257,67 @@ To load `hello_world` into the FPGA on the ChipWhisperer CW310 board follow the
 
 #### Troubleshooting
 
-If the firmware load fails, try pressing the "USB RST" button before loading the bitstream.
+If the firmware load fails, try pressing the "USR-RST" button before loading the bitstream.
 
 ## Connect with OpenOCD and debug
 
-To connect the ChipWhisperer CW310 FPGA board with OpenOCD, run the following command
+The CW310 supports JTAG-based debugging with OpenOCD and GDB via the standard ARM JTAG headers on the board (labeled USR Debug Headers).
+To use it, program the bitstream and bootstrap the desired firmware, then connect a JTAG adapter to one of the headers.
```

Comment:
Should the S2 switch next to the headers be set to 3.3V? or should it be set sense?

Reply:
Ah, good question--I added the answer to the doc. It should be set to 3.3 V.

We don't use the detection mechanism, and that wouldn't be the correct connection for the named standards in the schematics. For all the relevant ARM JTAG / SWD / Coresight specs, it's a voltage reference provided by the target to the debug adapter.

It'd be a different story if the FPGA were acting as the JTAG _debug adapter_, instead of the target (though you'd have to be careful about what you hook up). But that's not us! :)